mainapp/templates/account.html

```diff
@@ -59,6 +59,57 @@ <h4 class=" mb-0">
         </div>
     </div>
 
+    <!-- Tenant Apartments -->
+    {% if user.tenant_apartment_set.count > 0 %}
+        <hr>
+        <h1>Tenant's Apartment</h1>
```

Comment:
Could we change this heading to an `<h2>` or `<h3>`? The `<h1>` is awfully large. Also could the text be chanced to "Places I live"?

Same thing for the "Landlord's Apartments" heading, could the h-tag be made smaller and the text changed to "Places I own"

Reply:
<img width="963" alt="Screen Shot 2019-11-25 at 6 45 31 PM" src="https://user-images.githubusercontent.com/55497835/69587544-cfe9f380-0fb3-11ea-97ac-74ce23f3fe72.png">
I am getting this error while searching on local server. So I am unable to see the changes I am making.